website_menu_icon/static/src/xml/website.contentMenu.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+    <div t-name="website.contentMenu.entry.dialog.icon" t-extend="wysiwyg.widgets.link">
+        <t t-jquery="form" t-operation="append">
+            <div class="form-group row">
+                <label
+                    class="col-form-label col-md-3"
+                    for="o_icon_dialog_label_input"
+                >Icon / Image</label>
+                <div class="col-md-9 input-group ml-auto">
+                    <input
+                        type="text"
+                        name="image"
+                        id="o_icon_dialog_label_input_value"
+                        class="form-control o_we_search"
+                        placeholder="Search icon or image"
+                        readonly="1"
+                        t-att-value="widget.data.image"
+                    />
+                    <div class="input-group-append">
+                        <div class="input-group-text o_we_search_icon">
+                            <i
+                                class="fa fa-search"
+                                title="Search"
+                                role="img"
+                                aria-label="Search"
+                            />
+                        </div>
+                        <div class="input-group-text o_we_delete_icon">
+                            <i
+                                class="fa fa-remove"
+                                title="Remove"
+                                role="img"
+                                aria-label="Remove"
+                            />
+                        </div>
+                    </div>
+                </div>
+                <input
+                    type="text"
+                    name="image_type"
+                    id="o_icon_dialog_label_input_type"
+                    class="d-none"
+                    t-att-value="widget.data.image_type"
+                />
+            </div>
+        </t>
+    </div>
+    <!--
+    Had to copy and update the templates as I couldn't make the in-place extension work.
+    Also common mechanism didn't work for replacing t-call to "website.contentMenu.dialog.submenu.icon"
+-->
```

Comment:
Believe me, I've spent many hours trying to make work just that. Wanted the template neat and minimal. Specifically this example there:
`<t t-inherit="base.template" t-inherit-mode="extension">
    <xpath expr="//tr[1]" position="after">
        <tr><td>new cell</td></tr>
    </xpath>
</t>`
I wanted to use that on that sub-template to add the new stuff. Never got it working. Thought it was because it didn't have JS class representation. But it didn't work even for the main template. Was looking over sources of both 12.0 and 13.0 for use of this mechanism. It is just never used. In 14.0 it seems to be used widely but with OWL.

Any help/pointer here would be highly appreciated.